src/main/java/edu/umich/its/spe/SPEMaster.java

```diff
@@ -153,14 +160,65 @@ public void orchestrator () throws PersistBlobException, GradeIOException {
 			// Finish up, save data, send reports.
 			closeUpShop();
 		}
+
+
+	}
+
+	// Run the worker and then wait for a while before running again.
+	// Run then wait so will happen early the first time.
+	public void orchestrator() {
+
+		// get the interval wait time.
+		HashMap<String,String> testproperties = speproperties.getTest();
```

Comment:
What do "testproperties" mean here? is it not needed for the production run of SPE? Note: camel casing missing

Reply:
Property handling was revised and test properties is gone.